.github/pull_request_templates/feature-branch.md

```diff
@@ -0,0 +1,23 @@
+## Feature Branch: Title
+
+* Appendix doc:
+    - 
+* Link(s) to demo pages_ where this element can be viewed:
+    - 
+
+### Ready-for-merge Checklist
+
+- [ ] Expected files: all files in the merge request related to one feature request?
+- [ ] Is your branch up-to-date with the master branch?
```

Comment:
This is automated, so I think we can get by without asking it directly.  Branches cannot be merged if they aren't up to date, and github offers an "Update" button to do the update.  The UI makes it clear that this is necessary.

Reply:
@mwcz ack, i'll update that too, i need to rename the file to get github to "see" it